examples/savage/buttons_on_canvas.py

```diff
@@ -3,13 +3,14 @@
 import os.path
```

Comment:
clicking on this example from the examples UI window consistently crashes the demo application on my machine (Mac OS + Python 3.6 + PyQt)

the traceback is 

```
% edm run -e enable-test-3.6-pyqt-pillow -- python examples/demo.py
Qt has caught an exception thrown from an event handler. Throwing
exceptions from an event handler is not supported in Qt. You must
reimplement QApplication::notify() and catch all exceptions there.

libc++abi.dylib: terminating with uncaught exception of type kiva::$_0
```